### ISSUE TITLE
Some notes on inconsistent transliteration from Russian/Cyrillic

### ISSUE THREAD

**Issue by davidscotson** [standardebooks/john-reed_ten-days-that-shook-the-world#24]
I noticed the same newspaper name was transliterated two different ways while reviewing a concordance to look for spelling mistakes.

The author notes in the text:

> In the spelling of Russian names and words, I have made no attempt to follow any scientific rules for transliteration, but have tried to give the spelling which would lead the English-speaking reader to the simplest approximation of their pronunciation.

Even if restricting my attention just to names of cited newspapers, I think this is too big a can of worms for me to get involved in without experience, but I'll leave my findings here for anyone else to pick up on. Reminder, I know nothing about the Russian language:

- There appears to be no standard transliteration from Russian Cyrillic to English Latin characters, in fact there's multiple standards but wikipedia claims mostly things are done non-systematically.
- One that looks interesting to me is ISO 9:1995, which seems modern and uses diacritics to allow every character to map to a single character. This makes transliterated words stand out in a subtle way while still looking approachable in a way which I visually like, from the perspective of a modern ebook reader who knows English but no Russian/Cyrillic at all. But it's possibly intended more for scientific use, and wikipedia notes there were grumbles when this was used on Moscow street signs recently, so no idea if it makes sense in a historical epub text.
- I believe standardebooks has already updated some of the more recognizable names to their commonly used english variants e.g. "Trotzky -> Trotsky", which seems sensible when it applies, those are basically English words at that point like Moscow vs Moskva, but not sure of any systematic way to get a standard list of these, for the people, places and things mentioned in the book that could be used to look for potential items to update. (Russian names seems to be a whole other complicated thing in itself, see the git log on War and Peace, before you get into aliases and Lenin's name(s) seems different every time it's given in full in the text)
- The transliterations in the original scans of the book seem to be vaguely following one of the French transliteration systems (or possibly just an archaic english one) though not 100% consistent with itself.
- There seems to be a convention of using "ru-Latn" in xml:lang tags to specify that it's a romanisation and not Cyrillic characters being used, which might be useful here (https://tools.ietf.org/html/bcp47) and seems like a change that could be automated across the whole standardebooks catalogue if it seems helpful.

Specifying the actual transliteration system seems possible too, though it seems hard to track down a definitive list of what these should look like. These lang tags can get as complicated as `ru-Latn-s-Cyrl-t-iso9` (Russian-language text transliterated from the Cyrillic script into the Latin script via the ISO 9 transliteration system) but it's not clear anyone actually uses that system in practice or what benefits it would have for e-readers (potentially, things would be read out with more correct Russian pronunciation by software voices? Easier to look up translations/definitions if the dictionary recognizes that it can translate it into the canonical form?).

I might create some test branches, just converting the newspaper titles, and see how it performs visually in real e-reader software, and how software voices react.

All the different transliteration systems supported by this online tool:

https://www.translitteration.com/transliteration/en/russian/iso-9/

render the newspaper name above ( Биржевы́е ве́домости cut and pasted from Wikipedia) in the same way,  as Birževýe Védomosti, which is a good sign, though I've just noticed wikipedia gives it two slightly different variant names, that romanize differently)

**Comment by acabal** [comment]
As you noted there's lots of ways to transliterate Russian especially in the PD text era that we're working on. For that reason I don't think we're going to standardize on any one of them, but rather leave it as-is in the text outside of commonly-used names that would be recognizable today like Tolstoy or Trotsky. I agree that there's probably not a great systemic way to do that, except adding exceptions to modernize-spelling as we come across them. This mostly just requires a careful eye, at least a passing familiarity with Russian names/places, and a gut feeling as to whether or not a name is common enough in modern English to bother changing on a corpus-wide scale.

Changing tags to ru-Latn *might* be interesting but probably not worthwhile in the long run. Presumably a screen reader can already see that the text in a tag is in Latin letters and not Cyrillic so does specifying Latn change anything?

**Comment by davidscotson** [comment]
I did some experiments on different items in the text (place names and newspapers mostly) and the results were mixed. Probably the best example is the one I started on, Birževýe Védomosti, which looks vaguely exotic while still being equally approachable to an English reader. But others become more challenging, Like Novaya -> Novaā, which seems less readable/pronounceable to me. Many others came out unchanged. So on balance, leaving them as they are seems best unless it's a famous person/place that used to have multiple transliterations and modern society has standardized on a different one.

I also figured out that the reason many of the historical items had two spellings in the Russian Wikipedia is because they had a spelling reform around this same time, which at least the tool I was using just ignored and left the old character in place.

Potentially there's pronunciation or accessability benefits for software text but that's not really my area so I'll pass.

I'll note that I found mention of at least one old bug (in Firefox) where it was trying to use a Cyrillic font for transliterated text marked up as Ru which was visually sub-optimal, but it was old and presumably such issues have generally been worked out by now.
